platform/darwin/src/MGLStyle.h

```diff
@@ -177,6 +177,13 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  Returns a layer that conforms to `MGLStyleLayer` if any layer with the given
  identifier was found.
+ 
+ @note Layer identifiers are subject to change and are not guaranteed to exist
```

Comment:
This is an awkward place to put “subject to change”. Maybe in the next sentence, say that the default style’s layers are subject to change.